apis/v1/gateway_types.go

```diff
@@ -602,8 +593,27 @@ type ListenerTLSConfig struct {
 	Options map[AnnotationKey]AnnotationValue `json:"options,omitempty"`
 }
 
-// GatewayTLSConfig specifies frontend tls configuration for gateway.
+// GatewayTLSConfig specifies frontend and backend tls configuration for gateway.
 type GatewayTLSConfig struct {
+
+	// BackendTLS describes TLS configuration for gateway when connecting
+	// to backends.
+	// Support: Core
+	//
+	// +optional
+	// <gateway:experimental>
+	BackendTLS *GatewayBackendTLS `json:"backendTls,omitempty"`
```

Comment:
Since this is already in `tls`, recommend `tls.backendTLS` -> `tls.backend` 
```suggestion
	BackendTLS *GatewayBackendTLS `json:"backend,omitempty"`
```

Reply:
done